src/region.js

```diff
@@ -7,6 +7,8 @@
 // http://lostechies.com/derickbailey/2011/12/12/composite-js-apps-regions-and-region-managers/
 
 Marionette.Region = Marionette.Object.extend({
+  _cid: 'mnr',
```

Comment:
I think this could be `cidPrefix` 
https://github.com/jashkenas/backbone/blob/master/backbone.js#L393